app/src/main/java/fr/groggy/racecontrol/tv/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,115 @@
+package fr.groggy.racecontrol.tv.ui.home
+
+import android.content.res.Resources
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.Keep
+import androidx.fragment.app.viewModels
+import androidx.leanback.app.RowsSupportFragment
+import androidx.leanback.widget.*
+import dagger.hilt.android.AndroidEntryPoint
+import fr.groggy.racecontrol.tv.R
+import fr.groggy.racecontrol.tv.f1tv.Archive
+import fr.groggy.racecontrol.tv.ui.season.archive.SeasonArchiveActivity
+import fr.groggy.racecontrol.tv.ui.season.browse.SeasonBrowseActivity
+import org.threeten.bp.Year
+
+
+@Keep
+@AndroidEntryPoint
+class HomeFragment : RowsSupportFragment(), OnItemViewClickedListener {
+
+    private val listRowPresenter = ListRowPresenter(FocusHighlight.ZOOM_FACTOR_NONE).apply {
+        shadowEnabled = false
+        selectEffectEnabled = false
+    }
+    private val archivesAdapter = ArrayObjectAdapter(listRowPresenter)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupUIElements()
+        setupEventListeners()
+        buildRowsAdapter()
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        val view = super.onCreateView(inflater, container, savedInstanceState)
+        val params = view!!.layoutParams as ViewGroup.MarginLayoutParams
+        val resources: Resources = inflater.context.resources
+
+        val dimensionPixelSize = resources.getDimensionPixelSize(R.dimen.lb_browse_rows_fading_edge)
```

Comment:
Just use these `resources` directly without add it to a variable...

Like...
inflater.context.resources.resources.getDimensionPixelSize